File: benchmarks/interpreter/bench_compute.py
def fib(n):
    if n < 2: return n
    return fib(n-1) + fib(n-2)
s = 0
for i in range(1, 100001): s += i
print(f"sum: {s}")
print(f"fib(25): {fib(25)}")

File: benchmarks/interpreter/bench_compute.pl
sub fib { my $n = shift; return $n if $n < 2; return fib($n-1) + fib($n-2); }
my $sum = 0; for my $i (1..100000) { $sum += $i; }
print "sum: $sum\n";
print "fib(25): " . fib(25) . "\n";
